kubernetes/docs/try-with-minikube.md

```diff
@@ -0,0 +1,60 @@
+# Try with Minikube
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [Kubernetes](https://kubernetes.io/)
+- [Minikube](https://github.com/kubernetes/minikube)
+- [Helm](https://helm.sh/)
+
+Please follow
+[instructions](https://kubernetes.io/docs/tasks/tools/install-minikube/) to
+install Minikube.
+
+## Create/start a minikube cluster
+
+```sh
+minikube start -p shopping-cluster
+```
+
+## Build docker images for shopping and recommender for the minikube cluster
+
+```sh
+eval $(minikube -p shopping-cluster docker-env)
+npm run docker:build
+```
+
+## Configure kubectl
+
+```sh
+kubectl config use-context shopping-cluster
+```
+
+## Set up helm
+
+https://helm.sh/docs/using_helm/
+
+```sh
+kubectl create serviceaccount --namespace kube-system tiller
+kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin
```

Comment:
Please make this 1 line instead of 2

e.g.

kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller

Reply:
That's auto formatted.